### PR TITLE
WIP: update gap2seq to 3.1

### PR DESCRIPTION
**Pull request by notestaff**
:information_source:
Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).

----------------

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by dpryan79** [comment]
Nicely done @notestaff, do you want to merge this?

**Comment by notestaff** [comment]
Not quite yet, I have some more updates.  It doesn't need a runtime dependence on boost.  Also, I'm seeing some bugs in the tool, and want to see if I can patch them first.  But thanks for reviewing the PR.

File: recipes/gap2seq/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,53 @@
...
+    - coreutils
+    - binutils
   host:
     - boost
     - zlib
+    - python
+    - htslib
   run:
     - boost
     - zlib
```
Comment (bgruening) [inline recipes/gap2seq/meta.yaml]: zlib can also be removed as it has run_exports defined

**Comment by dpryan79** [comment]
@notestaff This is ready for merging afaik, let me know if it's otherwise ready.

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Pull request merged**